server/routes/index.test.ts

```diff
@@ -25,6 +27,19 @@ describe('GET /', () => {
         expect(res.text).toContain('Book a visit')
         expect(res.text).toContain('Change a visit')
         expect(res.text).toContain('View visits by date')
+        expect(res.text).toContain('Change establishment')
```

Comment:
All looks good - I'd just be tempted to take advantage of the `data-test` attribute (`data-test="change-establishment"`) you added for this element to make checking for this a bit more targeted  here and in the other tests in this PR. Then you can be sure you've found this element rather than happening to find a match anywhere on the page.

Reply:
☝️ 😄